api/v1alpha/src/accounts/dataManager.js

```diff
@@ -279,7 +279,6 @@ async function createOrUpdateAccount(projectId, accountId, data) {
 async function getAccount(projectId, accountId, email, emailType) {
     const table = getTableFqdn(projectId, cfg.cdsDatasetId, cfg.cdsAccountViewId);
     const fields = Array.from(cfg.cdsAccountViewFields).join();
-    const limit = 1;
```

Comment:
clean up, the limit would eliminate the possibility of if statement catching scenario of multiple entries - would prefer it throws an exception rather than allow duplicate records to carry on.

Reply:
Should there be a safe guard against returning too many rows though? e.g. increase the limit to 100?

Reply:
Great point, we only care to know if there are duplicates or greater. So we can set the limit to 2.